internal/factsengine/gatherers/sbd.go

```diff
@@ -50,7 +48,7 @@ func (g *SBDGatherer) Gather(factsRequests []entities.FactRequest) ([]entities.F
 	facts := []entities.Fact{}
 	log.Infof("Starting SBD config Facts gathering")
 
-	conf, err := loadSbdConf(g.configFile)
+	conf, err := loadSbdConfig(g.configFile)
```

Comment:
Here is where we should put the error handling in my opinion.
Directly here, one one error, and that's all